utils/mil_tools/nodes/online_bagger.py

```diff
@@ -47,7 +47,16 @@ def __init__(self):
                                              self.start_bagging)
 
         self.subscribe_loop()
-        rospy.loginfo('subscriber list: %s', self.subscriber_list)
+
+        rospy.loginfo('')
+        rospy.loginfo('Remaining Failed Topics:')
+        self.print_subscriber_list(False)
+        rospy.loginfo('')
+
+    def print_subscriber_list(self, status):
+        for topic in self.subscriber_list.keys():
+            if self.subscriber_list[topic][1] == status:
+                rospy.loginfo('{}, {}'.format(self.subscriber_list[topic], topic))
 
     def get_params(self):
```

Comment:
It feels like this function should have a docstring. Are there any parameters that are necessary (don't have defaults)?